tests/unit/models/ObjectLockConfiguration.js

```diff
@@ -178,3 +246,14 @@ describe('ObjectLockConfiguration class getValidatedObjectLockConfiguration',
         });
     });
 });
+
+describe('ObjectLockConfiguration class getConfigXML', () => {
+    passTestsGetConfigXML.forEach(test => {
+        const { config, description, expectedXml } = test;
+        it(`should return correct XML ${description}`, done => {
+            const responseXml = ObjectLockConfiguration.getConfigXML(config);
+            assert.strictEqual(responseXml, expectedXml);
+            done();
```

Comment:
minor point: since it's synchronous you don't need a `done` function, you can declare it without parameter as `() => {...}` and not call `done()` at the end.

Reply:
you're right; updated 👍